doc/howto-devel.rst

```diff
@@ -15,14 +15,33 @@ each supported distribution::
 Vagrant
 -------
 
-**Note**: this requires installing the `vagrant-sshfs`_ plugin.
+Fork this github repostiory located at `<https://github.com/proot-me/proot>`_
+
+``git clone [YOUR_REPOSITORY_URL]`` Clone the repository on your local machine
```

Comment:
I'm fine with assuming the developer already knows how to create a fork. We don't really need to add this.